client/landing/stepper/declarative-flow/new-hosted-site-flow.ts

```diff
@@ -83,6 +84,7 @@ const hosting: Flow = {
 						return navigate( 'trialAcknowledge' );
 					}
 
+					setSignupCompleteFlowName( flowName );
```

Comment:
The complete flow name is reset outside of this flow. We need it set so that new site is not created.